website/docs/r/data_lake_store.html.markdown

```diff
@@ -37,6 +44,16 @@ The following arguments are supported:
 
 * `tier` - (Optional) The monthly commitment tier for Data Lake Store. Accepted values are `Consumption`, `Commitment_1TB`, `Commitment_10TB`, `Commitment_100TB`, `Commitment_500TB`, `Commitment_1PB` or `Commitment_5PB`.
 
+* `encryption_state` - (Optional) Is Encryption enabled on this Data Lake Store Account? Possible values are `Enabled` or `Disabled`. Defaults to `Enabled`.
+
+* `encryption_type` - (Optional) The Encryption Type used for this Data Lake Store Account. Defaults to `SystemManaged` which is the only supported value at this time.
```

Comment:
Does the API default this? It's not defaulted in the schema. https://github.com/terraform-providers/terraform-provider-azurerm/pull/1623/files#diff-5741558d3f6a1e548c0a362adc779447R67

Reply:
It depends on if Encryption's enabled or disabled, will push an update